New Bank/NewBank/newbank/server/NewBankClientHandler.java

```diff
@@ -11,28 +11,22 @@
 	private NewBank bank;
 	private BufferedReader in;
 	private PrintWriter out;
+	private Boolean loggedIn;
 	
 	
 	public NewBankClientHandler(Socket s) throws IOException {
 		bank = NewBank.getBank();
 		in = new BufferedReader(new InputStreamReader(s.getInputStream()));
 		out = new PrintWriter(s.getOutputStream(), true);
+		loggedIn = false;
 	}
 	
 	public void run() {
 		// keep getting requests from the client and processing them
+		while(!loggedIn) {
+			loggedIn = loggedIn();
+		}
 		try {
-			String responce = "FAIL"; // initialise responce string
-			while (responce!="SUCCESS"){ // while loop until success is achieved, will not bring up login entry until then
-				// ask if login or signup
-				out.println("LOGIN or SIGNUP");
-				String request = in.readLine();
-				responce = bank.processRequest(request);
-				out.println(responce);
-			}
-		} catch (IOException | NumberFormatException e) {
-			e.printStackTrace();
-		} try {
```

Comment:
Moved try/catch to it's own method -> loggedIn() so that run() can be called recursively and use a boolean for logged in or logged out.